core-java-modules/core-java-streams-4/src/test/java/com/baeldung/streams/intarraytostrings/IntArrayToStingUnitTest.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.streams.intarraytostrings;
+
+import static com.baeldung.streams.intarraytostrings.ArrayConversionUtils.convertToString;
+import static com.baeldung.streams.intarraytostrings.ArrayConversionUtils.convertToStringArray;
+import static com.baeldung.streams.intarraytostrings.ArrayConversionUtils.convertToStringArrayWithBoxing;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class IntArrayToStingUnitTest {
```

Comment:
Let's correct the typo in the class/file name: IntArrayToStringUnitTest (you're missing the 'r' in 'String')